scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -2,7 +2,7 @@
 
     <div class="modal__header modal__header--flex">
         <h3 class="modal__heading" ng-show="!desk.edit._id" translate>Add New Desk</h3>
-        <h3 class="modal__heading" ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
+        <h3 class="modal__heading" ng-show="desk.edit._id" translate translate-params-author="{{desk.edit.name}}">Edit "{{desk.edit.name}}" Desk</h3>
```

Comment:
this translation is still bad - check example [in the docs](https://angular-gettext.rocketeer.be/dev-guide/translate-params/) on how to use `translate-params`